docs/DOCKER.md

```diff
@@ -129,6 +129,12 @@ To build the image using the most current tag:
 ⛰  docker build . --build-arg checkout=$(git describe --tags `git rev-list --tags --max-count=1`) -t myrepository/lnd:latest-tag
 ```
 
+or
+
+```shell
+⛰  make docker-release tag=$(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
AFAIK this doesn't build any docker images, it just builds the binaries within a container.